mopidy_spotify/translator.py

```diff
@@ -214,6 +214,7 @@ def to_playlist(
     bitrate=None,
     as_ref=False,
     as_items=False,
+    check_playable=True,
```

Comment:
The functionality of this new parameter is not tested.

Reply:
check_playable is passed straight through to web_to_track_refs, which is tested in https://github.com/mopidy/mopidy-spotify/blob/master/tests/test_translator.py#L454

Reply:
added a test for `web_to_track`, as well as testing the pass-though for `to_playlist`